### PR TITLE
fix(multiframe encapsulated): take slice of array buffer to worker for decoding

### PR DESCRIPTION
**Pull request by sedghi**
### Context

WADOURI multiframe was slow for the compressed images since the `getEncapsulatedPixelData` was returning the full array buffer and that was too big for copying over to the worker.

### Testing

Test this data in the local example https://ohif-assets.s3.us-east-2.amazonaws.com/multiframe-wadouri/compressed.dcm

Fixes https://github.com/cornerstonejs/cornerstone3D/issues/842
Fixes https://github.com/cornerstonejs/cornerstone3D/issues/843

#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

- [] I have run the `yarn build:update-api` to update the API documentation, and have
  committed the changes to this PR. (Read more here https://www.cornerstonejs.org/docs/contribute/update-api)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] "OS: 
- [] "Node version: 
- [] "Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Review by wayfarer3130 — CHANGES_REQUESTED** [review]
Just add the check for # of fragments / # of frames.   Might want to add that in a unit test so there is an obvious spot to add new tests to actually handle mismatches between # of frames  and fragments.

**Review by jbocce — COMMENTED** [review]
The changes look good. Perhaps address @wayfarer3130's comment. I tried restarting the Netlify build for the preview, but it failed again. If we can fix that too then I can give it a quick test. Thanks.

File: packages/core/src/init.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+console.debug("core: link: init: 7DF8FBDE-F3DB-4B91-83DC-A47AE03BDD73");
```
Comment (jbocce) [inline packages/core/src/init.ts:1]: Do we need this?

File: packages/dicomImageLoader/src/imageLoader/registerLoaders.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+console.debug("dicom loader: link: init: 7DF8FBDE-F3DB-4B91-83DC-A47AE03BDD73");
```
Comment (jbocce) [inline packages/dicomImageLoader/src/imageLoader/registerLoaders.ts:1]: Do we still want/need this?

**Review by jbocce — APPROVED** [review]
Thanks for the testing notes. I checked all of the various associated issues. Looks great!

**Pull request merged**